packages/genui/openui/README.md

```
@@ -0,0 +1,214 @@
+| Option            | Type                 | Default       | Description                                    |
+| ----------------- | -------------------- | ------------- | ---------------------------------------------- |
+| `root`            | `string`             | `'Stack'`     | Name of the root component                     |
+| `components`      | `DefinedComponent[]` | `[]` (merged) | Additional components appended to built-in set |
+| `componentGroups` | `ComponentGroup[]`   | `[]` (merged) | Additional groups appended to built-in groups  |
+
+## OpenUI DSL Syntax
+
+The DSL uses a **functional notation** (not XML). Each statement assigns
+a component instance to an identifier:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)